lib/Makefile

```diff
@@ -29,7 +29,7 @@ CFLAGS			+= -I. -DLTPLIB
 ifneq ($(ANDROID),1)
 FILTER_OUT_DIRS		+= android_libpthread android_librt
```

Comment:
Hello, I didn't see your message on the mailing list. You're right that they're missing and with how many new tests are making use of libpthread it would be better to correctly add the stub libraries. I'll modify this PR, thanks for catching that.